pyk/src/pyk/ktool/utils.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING, final
+
+from ..utils import run_process_2
+
+if TYPE_CHECKING:
+    from typing import Final
+
+
+@final
+@dataclass(frozen=True)
+class KDistribution:
```

Comment:
Could this maybe get a docstring as it's a top-level tool?

Reply:
Done.